src/main/java/org/openstreetmap/atlas/checks/distributed/IntegrityCheckSparkJob.java

```diff
@@ -206,24 +207,24 @@ public void start(final CommandMap commandMap)
 
         // Read priority countries from the configuration
         final List<String> priorityCountries = checksConfiguration
-                .get("priority.countries", Arrays.asList("BLR", "CHN", "CHL", "DNK", "UKR"))
-                .value();
+                .get("priority.countries", Collections.EMPTY_LIST).value();
 
         // Create a list of Country to Check tuples
         // Add priority countries first if they are supplied by parameter
         final List<Tuple2<String, Set<BaseCheck>>> countryCheckTuples = new ArrayList<>();
-        countries.stream().filter(priorityCountries::contains)
-                .forEach(country -> countryCheckTuples.add(new Tuple2<>(country, checksToExecute)));
+        countries.stream().filter(priorityCountries::contains).forEach(country -> countryCheckTuples
+                .add(new Tuple2<>(country, checksToExecute.get(country))));
 
         // Then add the rest of the countries
         countries.stream().filter(country -> !priorityCountries.contains(country))
-                .forEach(country -> countryCheckTuples.add(new Tuple2<>(country, checksToExecute)));
+                .forEach(country -> countryCheckTuples
+                        .add(new Tuple2<>(country, checksToExecute.get(country))));
 
         // Log countries and integrity
         logger.info("Initialized countries: {}", countryCheckTuples.stream().map(tuple -> tuple._1)
                 .collect(Collectors.joining(",")));
-        logger.info("Initialized checks: {}", checksToExecute.stream().map(BaseCheck::getCheckName)
-                .collect(Collectors.joining(",")));
+        logger.info("Initialized checks: {}", checksToExecute.values().stream().flatMap(Set::stream)
+                .map(BaseCheck::getCheckName).distinct().collect(Collectors.joining(",")));
```

Comment:
This log message is not as useful any more. Due to overrides `checksToExecute` set is in a different state.

Reply:
This will still log all the checks loaded for at least one country. Which I agree is no longer that useful, but what would be a good alternative? Should we log at each county distributed job?